website/components/docs-page/style.css

```diff
@@ -46,7 +48,9 @@
     /* TODO: this should be applied in global styles, temporary override here */
     & pre,
     & code {
-      font-size: 0.875em;
+      @media screen {
+        font-size: 0.875em;
+      }
```

Comment:
Can we move these changes over to the print stylesheet so that this PR is purely additive?